src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlDateTime2.java

```diff
@@ -48,7 +47,19 @@ public SqlDateTime2() {
     public Object createdata() {
         Timestamp temp = new Timestamp(ThreadLocalRandom.current().nextLong(((Timestamp) minvalue).getTime(), ((Timestamp) maxvalue).getTime()));
         temp.setNanos(0);
-        String timeNano = temp.toString().substring(0, temp.toString().length() - 1) + RandomStringUtils.randomNumeric(this.precision);
+        
+        Random rnd = new Random();
+        // this will generate a number that is this.precision number of digits.
+        int randomNumeric;
+        if (this.precision <= 0) {
+            randomNumeric = 0;
+        } else if (this.precision == 1) {
+            randomNumeric = rnd.nextInt(10);
+        } else {
+            randomNumeric = (int) Math.pow(10, this.precision - 1) + rnd.nextInt((int) (9 * Math.pow(10, this.precision - 1)));
```

Comment:
can we save this to a var so we don't have to call Math.pow(10, this.precision - 1) twice?